app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new(session[:user_params])
+    session.delete(:user_params)
```

Comment:
What is the purpose of the delete.session? Test suite passes without it.

Reply:
I only want to store that user data in the session for as long as it is needed.

If the user successfully registers, then that data is saved into the database. 
In this case, we already have it saved, so we can delete it from the sesson.

It adds an extra layer of security -> in a later commit, I actually limit the params that are passed into the sesson.

Reply:
```
  def user_registration_params
    params.permit(:name, :address, :city, :state, :zip, :email)
  end
```